src/style/core.ts

```diff
@@ -38,6 +38,11 @@ const root = css`
   --rdg-selection-color: #66afe9;
   --rdg-font-size: 14px;
 
+  position: relative;
+  display: grid;
+  grid-template-columns: var(--rdg-template-columns);
+  grid-template-rows: var(--rdg-template-rows);
```

Comment:
It looks like we only use these variables once now, let's use the css properties directly instead.